src/Mgate/SuiviBundle/Entity/EtudeRepository.php

```diff
@@ -24,6 +24,7 @@ class EtudeRepository extends EntityRepository
 {
     public function findByNumero($numero)
     {
+        /*
```

Comment:
No parameter escaping !! :-1: 

BTW that function is unused in the project. Thus it can be completely removed, instead of re-implementing the default findByXX doctrine behaviour.